src/Dropdown/index.jsx

```diff
@@ -43,10 +43,10 @@ class Dropdown extends React.Component {
     }
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps, prevState) {
     if (this.state.open) {
       this.menuItems[this.state.focusIndex].focus();
-    } else if (this.toggleElem) {
+    } else if (prevState.open && this.toggleElem) {
```

Comment:
More of a clarifying question, but why are we checking the previous `open` state here instead of the current `open` state?

Reply:
@adamstankiewicz we already defining `this.state.open` behavior in above `if` condition.
I believe [this.toggleElem.focus();](https://github.com/edx/paragon/blob/f9076fbc0d8d8b167c9c93d2038435339b4bcac7/src/Dropdown/index.jsx#L50) should run only if we just closed the `Dropdown`, to focus the `Button` Again. 
that is way we are adding extra check `prevState.open`